ts/translations-list/translations-list.directive.ts

```diff
@@ -0,0 +1,139 @@
+module lui.translationslist {
+	"use strict";
+
+	class LuidTranslationsList implements angular.IDirective {
+		public static IID: string = "luidTranslationsList";
+		public restrict: string = "E";
+		public templateUrl: string = "lui/templates/translations-list/translations-list.html";
+
+		public require: string[] = ["ngModel", LuidTranslationsList.IID];
+		public scope = {
+			mode: "@", // Allowed values: "lucca",
+			disabled: "=ngDisabled", // enable editing
```

Comment:
i dont like you using `ngDisabled`, its an angular directive that adds `disabled="disabled"`, dont try to piggy back on it

also dont use only `disabled` as an attribute name cuz ie 11- disables js events on children elements of a DOM element that has `disable="whatever"`

you can disable your input via css by using `[disable="disabled"] css selector on your root element